tests/test_wasm_builtin_simd.c

```diff
@@ -957,26 +969,26 @@ int EMSCRIPTEN_KEEPALIVE __attribute__((__optnone__)) main(int argc, char** argv
     i8x16_shl((i8x16){0, 1, 2, 4, 8, 16, 32, 64, -128, 3, 6, 12, 24, 48, 96, -64}, 1),
     ((i8x16){0, 2, 4, 8, 16, 32, 64, -128, 0, 6, 12, 24, 48, 96, -64, -128})
   );
-  expect_vec(
-    i8x16_shl((i8x16){0, 1, 2, 4, 8, 16, 32, 64, -128, 3, 6, 12, 24, 48, 96, -64}, 8),
-    ((i8x16){0, 1, 2, 4, 8, 16, 32, 64, -128, 3, 6, 12, 24, 48, 96, -64})
-  );
+  /* expect_vec( */
+  /*   i8x16_shl((i8x16){0, 1, 2, 4, 8, 16, 32, 64, -128, 3, 6, 12, 24, 48, 96, -64}, 8), */
+  /*   ((i8x16){0, 1, 2, 4, 8, 16, 32, 64, -128, 3, 6, 12, 24, 48, 96, -64}) */
+  /* ); */
   expect_vec(
```

Comment:
the description says "add", why were some tests commented out?

Reply:
Sorry, I should have elaborated more. I will in the commit message. The SIMD tests were entirely disabled in #9241 due to a V8 bug with the shifts. The V8 bug has not been fixed yet, but I wanted the tests to run again, so I have commented out the failing tests.

Reply:
Sounds good, thanks!